control-plane/stor-port/src/types/v0/store/volume.rs

```diff
@@ -519,6 +509,11 @@ impl SpecTransaction<VolumeOperation> for VolumeSpec {
                 VolumeOperation::Resize(size) => {
                     self.size = size;
                 }
+                VolumeOperation::SetVolumeProperty(property) => match property {
+                    VolumeProperty::MaxSnapshots(max_snapshots) => {
+                        self.max_snapshots = Some(max_snapshots);
```

Comment:
Probably at some point it'll be time to refactor the volume spec, and tidy up things, such as putting all properties together etc

Reply:
Yes, all such properties can be put in an enum in volume spec.